### PR TITLE
Added read of map_d from MTL files

### PR DESCRIPTION
**Pull request by NissimHadar**
# Description
The previous code was not reading the map_d field in MTL files (part of the WaveFront OBJ specification).
# Test
A test is available here: https://github.com/highfidelity/hifi_tests/blob/master/tests/content/entity/model/modelReaders/objReader/test.md

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12414/HighFidelity-Beta-PR13883-62814de.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12414/HighFidelity-Beta-PR13883-62814de.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12414/HighFidelity-Beta-Interface-PR13883-62814de.dmg

File: libraries/fbx/src/OBJReader.cpp [hunk 1]
```diff
@@ -263,15 +263,19 @@ void OBJReader::parseMaterialLibrary(QIODevice* device) {
...
+                                     " shininess:" << currentMaterial.shininess << 
+                                     " opacity:" << currentMaterial.opacity <<
+                                     " diffuse color:" << currentMaterial.diffuseColor << 
+                                     " specular color:" << currentMaterial.specularColor << 
+                                     " emissive color:" << currentMaterial.emissiveColor << 
+                                     " diffuse texture:" << currentMaterial.diffuseTextureFilename << 
+                                     " specular texture:" << currentMaterial.specularTextureFilename << 
+                                     " emissive texture:" << currentMaterial.emissiveTextureFilename << 
+                                     " bump texture:" << currentMaterial.bumpTextureFilename;
+                                     " opacity texture:" << currentMaterial.opacityTextureFilename;
```
Comment (sabrina-shanman) [inline libraries/fbx/src/OBJReader.cpp]: Syntax error here. Second to last line should have "<<" instead of  ";"

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12558/HighFidelity-Beta-PR13883-d99cd2a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12558/HighFidelity-Beta-Interface-PR13883-d99cd2a.dmg

**Comment by marko8904** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12565/HighFidelity-Beta-PR13883-d99cd2a.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12565/HighFidelity-Beta-PR13883-d99cd2a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12565/HighFidelity-Beta-Interface-PR13883-d99cd2a.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Test passes.

**Pull request merged**